standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2451,19 +2497,24 @@ service ThriftHiveMetastore extends fb303.FacebookService
   void drop_catalog(1: DropCatalogRequest catName) throws (1:NoSuchObjectException o1, 2:InvalidOperationException o2, 3:MetaException o3)
 
   void create_database(1:Database database) throws(1:AlreadyExistsException o1, 2:InvalidObjectException o2, 3:MetaException o3)
```

Comment:
shouldn't this be deleted?

Reply:
I thought we are planning to keep the backward compatibility with the old client and a new server for at least one release. So that's the reason why I'm keeping the deprecated APIs. Should I just delete them?

Reply:
Is there only one `create_database_req` method for creating `database` stuff in the future release?